llvm/test/CodeGen/WebAssembly/strip-tls.ll

```diff
@@ -0,0 +1,28 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 4
```

Comment:
I assume we already have one or more tests for TLS stripping.. should we give this one a more specific name?  (or merge them?)

Reply:
Well none of the test outputs changed with my patch, so it seemed better to create a new one.

It's likely because the existing tests don't use `llvm.threadlocal.address`. Not sure if someone should update the tests to use it but seemed out of scope here...

Reply:
I guess I can try to convert some of the existing TLS tests to use `llvm.threadlocal.address` instead of creating a new test...

Reply:
llvm/test/CodeGen/WebAssembly/tls-local-exec.ll and tls-general-dynamic.ll cover TLS stripping, but they don't use `llvm.threadlocal.address`. It's probably worth updating those to reflect the IR currently generated by clang, but I think it makes sense to land this with the new test and then clean up later.

Reply:
Right, I was mostly suggesting that maybe this test be called something more like `trip-tls-intrinsics.ll` or something like that (i'm not sure that the right term is for `llvm.threadlocal.address`).

My as you can tell I don't fully understand this code no need to change if you think this is fine.

Reply:
Actually changed `tls-local-exec.ll` and `tls-general-dynamic.ll` and verified the changed versions do indeed crash without this fix. (And only now realize derek didn't actually ask to change them right now, but I guess it won't hurt :) )

Reply:
I still appreciate it :)